recipes/fd/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "7.3.0" %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: fd
```

Comment:
There is a unix utility with the same name. In debian in the fdclone package. Maybe we should change the name of the package?

Reply:
Well, that is the problem of two letters only... Renaming the package or command would make it cumbersome to find it and would also then deviate from the documentation or how it is already named in e.g. homebrew, https://github.com/Homebrew/homebrew-core/blob/master/Formula/fd.rb

If you have a good idea here, feel free to update the PR.

Reply:
Looking around at other package managers it seems like the consensus is either `fd` (Homebrew, Arch, openSUSE, Slackware) or `fd-find` (Ubuntu, Fedora, Debian, Free BSD).

Reply:
@chrisburr Thanks for looking that all up! Naming the packing fd-find sounds like a good idea. Will rename it after being back from afk (or I manage to do so with working copy on a tiny iPhone). Any other findings? Thanks

Reply:
Nope it looks good, I've edited the name so LGTM 👍